chromadb/api/fastapi.py

```diff
@@ -434,6 +438,15 @@ def get_settings(self) -> Settings:
         """Returns the settings of the client"""
         return self._settings
 
+    @property
+    @override
+    def max_batch_size(self) -> int:
+        resp = self._session.get(self._api_url + "/pre-flight-checks")
```

Comment:
why are we making the request everytime? If we have it cached, we can safely ignore the get?

Reply:
I thought I'd fixed it. Fixed now.